darshan-util/pydarshan/darshan/templates/summary.html

```diff
@@ -0,0 +1,384 @@
+        </h4>
+    </div>
+    <div id="collapse4" class="panel-collapse collapse in">
+        <div class="panel-body">
+            <div class="container">
+                <div class="row">
+                    <div class="col-lg-12">
+                        <h3>Variance in Shared File (POSIX and STDIO)</h3>
+                        <table class="table">
+                            <thead>
+                                <tr>
+                                    <th rowspan="2">File Suffix</th>
+                                    <th rowspan="2">Processes</th>
+                                    <th colspan="3">Fastest</th>
+                                    <th colspan="3">Slowest</th>
+                                    <th colspan="2">&sigma;</th>
+                                </tr>
+                                <tr>
+                                    <th>Rank</th>
+                                    <th>Time</th>
+                                    <th>Bytes</th>
+                                    <th>Rank</th>
+                                    <th>Time</th>
+                                    <th>Bytes</th>
+                                    <th>Time</th>
+                                    <th>Bytes</th>
+                                </tr>
+                            </thead>
+                            <tbody>
+                                <tr>
+                                    <td>blah.txt</td>
+                                    <td>100</td>
+                                    <td>1</td>
+                                    <td>2</td>
+                                    <td>3</td>
+                                    <td>0</td>
+                                    <td>4</td>
+                                    <td>5</td>
+                                    <td>6</td>
+                                    <td>7</td>
+                                </tr>
+                            </tbody>
+                        </table>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </div>
+</div>
+</div>
+</div>
+</div>
+
+<div class="panel panel-primary">
+    <div class="panel-heading">
+        <h4 class="panel-title">
+            <a data-toggle="collapse" href="#collapse1">APMPI</a>
+        </h4>
+    </div>
+    <div id="collapse1" class="panel-collapse collapse in">
+    <div class="panel-body">
+        <div class="container">
+        <div class="row">
+            <div class="col-lg-6">
+                    <img class="img-rounded" alt="APMPI-call-times" src="data:image/png;base64,${var.apmpi_call_time}"/>
+            </div>
+            <div class="col-lg-6">
+                    <img class="img-rounded" alt="APMPI-rank-totaltimes" src="data:image/png;base64,${var.apmpi_rank_totaltime}"/>
+            </div>
+        </div>
+        </div>
+     </div>
+     </div>
+</div>
+
+
+</body>
+</html>
```

Comment:
This will need a mention in MANIFEST.in

Reply:
Jakob is correct here--the code is not currently usable from a Python install (see below), I'll try to push in the fix...

```
Traceback (most recent call last):
  File "/home/tyler/github_projects/spack/opt/spack/linux-ubuntu18.04-skylake_avx512/gcc-7.4.0/python-3.7.4-p6llyclmnre4iur3olezociwacltbqpc/lib/python3.7/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/tyler/github_projects/spack/opt/spack/linux-ubuntu18.04-skylake_avx512/gcc-7.4.0/python-3.7.4-p6llyclmnre4iur3olezociwacltbqpc/lib/python3.7/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/home/tyler/python_37_venv/lib/python3.7/site-packages/darshan-3.3.0.3-py3.7.egg/darshan/cli/__main__.py", line 3, in <module>
    darshan.cli.main()
  File "/home/tyler/python_37_venv/lib/python3.7/site-packages/darshan-3.3.0.3-py3.7.egg/darshan/cli/__init__.py", line 160, in main
    mod.main(args)
  File "/home/tyler/python_37_venv/lib/python3.7/site-packages/darshan-3.3.0.3-py3.7.egg/darshan/cli/summary.py", line 636, in main
    template = loader.load('summary.html')
  File "/home/tyler/python_37_venv/lib/python3.7/site-packages/genshi/template/loader.py", line 248, in load
    raise TemplateNotFound(filename, search_path)
genshi.template.loader.TemplateNotFound: Template "summary.html" not found
```

Reply:
Ok, I pushed in the fix for this particular issue, so I'll mark it resolved...